share/spice/coderwall/coderwall.js

```diff
@@ -0,0 +1,62 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_coderwall = function(api_result){
+
+        // Check if API provided information
+        if ($.isEmptyObject(api_result.data)) {
+            return Spice.failed('coderwall');
+        }
+
+        Spice.add({
+            id: "coderwall",
+            name: "Coderwall",
```

Comment:
We're aiming to avoid brands as tabs, and instead generalize them into specific topics.

Let's change the `name` to `Social` for this IA.